jupyter_notebooks/exercises/GitPush_exercise3.ipynb

```diff
@@ -129,7 +129,7 @@
     "    plt.xlabel('Predicted')\n",
```

Comment:
<p>I needed 'pltxs' because I wanted to show the non processed signals in the fancy plot later on</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eigen-carmona/mlreview_notebooks/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>